janis_bioinformatics/tools/pmac/performanceSummaryGenomeWorkflow.py

```diff
@@ -78,3 +84,41 @@ def constructor(self):
         )
 
         self.output("performanceSummaryOut", source=self.performancesummary.out)
+
+    def tests(self):
+        with open(
+            os.path.join(
+                BioinformaticsTool.test_data_path(),
+                "wgsgermline_data",
+                "NA12878-BRCA1_performance_summary.csv",
+            ),
+            "r",
+        ) as f:
+            expected_content = f.read()
```

Comment:
If we intend to check exact match, I think we don't want to pass this to `TextFile.basic_test` as this is only doing partial match.
I think it would be better if we write out an additional `TTestExpectedOutput` using `TTestPreprocessor.FileContent` and `expected_file_path` pointing to `NA12878-BRCA1_performance_summary.csv`

Reply:
I've updated test for TextFile data type to allow it to take in a file path.